packages/ott-vis-panel/src/components/legend.css

```diff
@@ -0,0 +1,31 @@
+.legendBox {
+	position: absolute;
+	bottom: 0;
+	right: 0;
+	padding-bottom: 150px;
+	padding-right: 150px;
```

Comment:
This results in:
![image](https://github.com/dyc3/opentogethertube/assets/1808807/731c232f-4343-4633-bba3-9fa73d0b8f21)
which is not the best.

Offsetting the legend from the border would be better done setting `bottom` and `right` instead of adding padding.

Reply:
got it